src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -85,7 +83,8 @@ public abstract class AbstractTargetConfiguration implements TargetConfiguration
             "-H:+ReportExceptionStackTraces",
             "-H:-DeadlockWatchdogExitOnTimeout",
             "-H:DeadlockWatchdogInterval=0",
-            "-H:+RemoveSaturatedTypeFlows"
+            "-H:+RemoveSaturatedTypeFlows",
+            "--add-exports=org.graalvm.nativeimage.builder/com.oracle.svm.core.jdk=ALL-UNNAMED" // Required for GluonFeature
```

Comment:
GluonFeature is only used by the Linux target. You'll need to apply this argument only in that case, so I'd suggest adding another method like `getNativeImageArguments()` and override it from Linux.